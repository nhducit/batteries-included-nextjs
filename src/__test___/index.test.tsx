/* eslint-env jest */

import { render } from '@testing-library/react'
import React from 'react'
import App from '../pages/index'

describe('With React Testing Library', () => {
  it('Shows "Hello world!"', () => {
    const { getByText } = render(<App />)

    expect(getByText('Battery included Next.js')).not.toBeNull()
  })
})

describe('With React Testing Library Snapshot', () => {
  it('Should match Snapshot', () => {
    const { asFragment } = render(<App />)

    expect(asFragment()).toMatchSnapshot()
  })
})
