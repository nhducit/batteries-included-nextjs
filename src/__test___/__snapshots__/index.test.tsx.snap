// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With React Testing Library Snapshot Should match Snapshot 1`] = `
<DocumentFragment>
  <div>
    <nav>
      <ul
        class="flex justify-between items-center p-8"
      >
        <li>
          <a
            class="text-blue-500 no-underline"
            href="/"
          >
            Home
          </a>
        </li>
        <ul
          class="flex justify-between items-center"
        >
          <li
            class="ml-4"
          >
            <a
              class="btn-blue no-underline"
              href="https://github.com/zeit/next.js"
            >
              GitHub
            </a>
          </li>
          <li
            class="ml-4"
          >
            <a
              class="btn-blue no-underline"
              href="https://nextjs.org/docs"
            >
              Docs
            </a>
          </li>
        </ul>
      </ul>
    </nav>
    <div
      class="hero"
    >
      <h1
        class="title"
      >
        Battery included Next.js
      </h1>
      <p>
        env SECRET
        <strong />
      </p>
    </div>
  </div>
</DocumentFragment>
`;
